core/web/loader/loader_test.go

```diff
@@ -424,3 +425,44 @@ func TestLoader_SpecErrorsByJobID(t *testing.T) {
 		assert.ErrorIs(t, found[0].Error, sql.ErrNoRows)
 	})
 }
+
+func TestLoader_loadByEthTransactionID(t *testing.T) {
+	t.Parallel()
+
+	txmORM := &txmgrMocks.ORM{}
+	app := &coremocks.Application{}
+	ctx := InjectDataloader(testutils.Context(t), app)
+
+	defer t.Cleanup(func() {
+		mock.AssertExpectationsForObjects(t, app, txmORM)
+	})
```

Comment:
`t.Cleanup` already registers the func internally to be executed like a defer when the test completes, so the explicit `defer` can be omitted. However, I think we can go a step further in this case, and drop the defer/cleanup completely by using the mock constructors, which register this automatically:
```suggestion
	txmORM := txmgrMocks.NewORM(t)
	app := coremocks.NewApplication(t)
	ctx := InjectDataloader(testutils.Context(t), app)
```
https://github.com/smartcontractkit/chainlink/blob/39924684e1e61a1b7eb83a5ba1aecdc3e5a0dc67/core/chains/evm/txmgr/mocks/orm.go#L245-L249